src/sunburst/index.js

```diff
@@ -86,14 +86,24 @@ function buildLabels(mappedData) {
     const truedAngle = -1 * row.angle + Math.PI / 2;
     const truedAngle0 = -1 * row.angle0 + Math.PI / 2;
     const angle = (truedAngle0 + truedAngle) / 2;
+    const rotateLabels = !row.dontRotateLabel;
+    const rotAngle = -angle / (2 * Math.PI) * 360;
```

Comment:
No issue regarding ambiguous priority of operators? Maybe add parens to be sure?